recipe/meta.yaml

```
@@ -73,7 +73,7 @@ requirements:
     - shyaml
     - jq  # [unix]
     - requests
-    - ruamel_yaml
+    - ruamel.yaml
```

```suggestion
    - ruamel.yaml <0.19.0a0
```
would probably make sense. (No idea about what lower bound would be needed.)